src/main/kotlin/lotto/business/PrizeResults.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.business
+
+class PrizeResults(prizeCountMap: Map<LotteryPrize, Int>) {
+
+    private val _prizeCountMap = prizeCountMap.toMutableMap()
```

Comment:
가급적 프로퍼티와 메서드는 타입을 명시하도록 노력해보시면 어떨까요? 🤗

```suggestion
    private val _prizeCountMap: MutableMap<LotteryPrize, Int> = prizeCountMap.toMutableMap()
```


추가적으로 `_prizeCountMap` 프로퍼티는 불변 Collection인 Map으로 선언하셔도 무방했을 것 같은데,  가변 Collection으로 두신 이유가 있을까요?